core/mavlink_commands.h

```diff
@@ -53,30 +54,27 @@ class MavlinkCommands
     const MavlinkCommands &operator=(const MavlinkCommands &) = delete;
 
 private:
-    enum class State {
-        NONE,
-        WAITING,
-        IN_PROGRESS,
-        DONE,
-        FAILED
-    } _state = State::NONE;
-    std::mutex _state_mutex {};
+    static constexpr double DEFAULT_TIMEOUT_NORMAL_S = 0.5;
+    static constexpr double DEFAULT_TIMEOUT_IN_PROGRESS_S = 3.0;
+    static constexpr int RETRIES = 3;
 
     struct Work {
-        int retries_to_do = 3;
-        double timeout_s = 0.5;
+        int num_sent = 0;
```

Comment:
can we use a more descriptive name here? is `num_set` the number of times a command has been set after retries? I tried following the code and got a bit confused

Reply:
It is `num_sent`, not `num_set` but I've changed it to `num_command_sent` in a746eab.